go.mod

```diff
@@ -6,13 +6,13 @@ require (
 	github.com/RoaringBitmap/roaring v0.4.18
 	github.com/golang/protobuf v1.3.3-0.20190920234318-1680a479a2cf
 	github.com/google/uuid v1.3.0
-	github.com/pkg/errors v0.8.1
-	github.com/sirupsen/logrus v1.4.1
+	github.com/longhorn/backupstore v0.0.0-20211103081249-ebc3c90895ba
+	github.com/longhorn/longhorn-engine v1.2.4
```

Comment:
This is tricky, because generate_grpc.sh using the master of longhorn-engine, but here use v1.2.4. 

Would like to see Shuo's feedback.

Reply:
Updated to master(v1.3.0-preview1).

Reply:
IIRC, we plan to switch to the version-format image tag rather than the data-format tag for instance manager. Then in the future, we can have one-one matching. For now I think we need to use v1.3.0 longhorn-engine repo.